public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -39,6 +39,35 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 	</div>
 </div>
 
+<b>Connection Certificate Files</b>
```

Comment:
Are we sure we don't want the following forms to be part of the previous form group?

Reply:
i have no idea tbqh, i went looking for examples but the elasticsearch config partial isn't in the same place, and it appears most of the other connections also don't support TLS client verification.

there is also an argument that perhaps `SSL` should be `TLS` to be more,, modern?

Reply:
Since we use the term SSL elsewhere on this page (and also in the documentation you added for the new settings), doesn't it make sense to keep using that for consistency?

Reply:
If we have users who may not know if SSL and TLS are synonymous then it would be a good idea to use "SSL (TLS)" to make sure that everyone understands it.